docs/conditional.rst

```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
         views = NumberAttribute(default=0)
 
 
-AND vs. OR
-^^^^^^^^^^
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
 
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+PynamoDB supports creating condition expressions from attributes using a simple syntax. Attributes can be compared against values.
+Any value provided will be serialized using the serializer defined for that attribute.
 
 .. code-block:: python
 
-    thread_item = Thread('Existing Forum', 'Example Subject')
+    print(Thread.forum_name == 'Some Forum')
```

Comment:
I'm not sure I follow, this works under python 2.7:
```
(venv) jpinner-mbp121:PynamoDB jeffpinner (conditional-operations)$ python
Python 2.7.11 (default, Jun  3 2016, 13:12:55) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pynamodb.attributes import UnicodeAttribute
>>> forum_name = UnicodeAttribute(attr_name='forum')
>>> print(forum_name == 'Some Forum')
forum = Some Forum
```

As for pasting the result from the interpreter -- I was following the style of the existing examples in this file, and as far as I can tell, this gets render as a copyable code-block in the documentation, not as an interpreter snippet, so adding the results would prevent a user from copy-pasting.